README.md

```diff
@@ -1 +1,49 @@
-# java-jaxrs
\ No newline at end of file
+# OpenTracing JAX-RS Instrumentation
+
+Warning: This library is still a work in progress!
+
+[![Travis](https://travis-ci.org/opentracing-contrib/java-jaxrs.svg?branch=master)](https://travis-ci.org/opentracing-contrib/java-jaxrs)
+
+OpenTracing instrumentation for JAX-RS standard. 
+It supports server and client request tracing.
+
+## Tracing Server Requests
+```
+// register this in javax.ws.rs.core.Application
+ServerTracingDynamicFeature.Builder
+    .traceAll(yourPreferredTracer)
+    .withStandardTags()
+    .withDecorator(ServerSpanDecorator.HTTP_PATH_OPERATION_NAME)
+    .build();
+
+@GET
+@Path("/hello")
+@Traced(operationName = "helloRenamed") // optional, by default operation name is provided by ServerSpanDecorator
+public Response hello(@BeanParam CurrentSpan currentSpan) {
```

Comment:
CurrentSpan makes sense, and is something similarly needed in other async libraries. Ack that the means to inject it is framework specific, and it must be so because thread locals doesn't work in all cases.